plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoMetadata.java

```diff
@@ -120,6 +148,12 @@ public MongoTableHandle getTableHandle(ConnectorSession session, SchemaTableName
     public ConnectorTableMetadata getTableMetadata(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
         requireNonNull(tableHandle, "tableHandle is null");
+        MongoTableHandle handle = (MongoTableHandle) tableHandle;
+        if (isPassthroughQuery(handle)) {
```

Comment:
Since we capture FILTER to be applied on a collection - can we reuse the same column details of that of a collection ?